python/src/einsum_onnx.py

```diff
@@ -98,14 +176,14 @@ def make_constant_model(graph_name, output_name, tensor):
         )
     )
 
-def make_identity_node(input_name, output_name):
+def make_identity_node(input_name, output_name) -> OnnxNode:
     return onnx.helper.make_node(
         "Identity",
         inputs=[input_name],
         outputs=[output_name],
     )
 
-def run_model(model, *inputs):
+def run_model(model: OnnxModel, *inputs):
     sess = onnxruntime.InferenceSession(model.SerializeToString())
```

Comment:
when I run
```python3 -m mypy python/src/einsum_onnx.py```
on your patch I get this error:
```python/src/einsum_onnx.py:187: error: "OnnxModel" has no attribute "SerializeToString"```
plus three
```module is installed, but missing library stubs or py.typed marker```
errors which we could silence with `# type: ignore`

Reply:
Good catch! Fixed.